src/Angor/Client/Pages/Invest.razor

```diff
@@ -1048,7 +1049,8 @@ else
 
     public class InvestmentModel
     {
-        public decimal InvestmentAmount { get; set; }
+        public long InvestmentAmount { get; set; } // Amount in satoshis
```

Comment:
I propose that we get rid of the `InvestmentAmount` type here because it is just confusing.
We keep the amount in btc as user input and we convert to sats when we need to use it